src/frontend/src/invites.tsx

```diff
@@ -17,6 +35,56 @@ export const Invites = () => {
         loadInvites();
     }, []);
 
+    const update = async (code: string) => {
+        const updatedInvite = updatedInvites.find(
+            ({ code: inviteCode }) => code === inviteCode,
+        );
+        if (!updatedInvite) {
+            return;
+        }
+        return window.api
+            .call<any>(
+                "update_invite",
+                code,
+                updatedInvite.credits !== undefined &&
+                    updatedInvite.credits >= 0
+                    ? updatedInvite.credits
+                    : null,
```

Comment:
Wouldn't this work too here?
```suggestion
                updatedInvite.credits || null
```

Reply:
0 is valid number while null means empty.